Octokit/Http/SimpleJsonSerializer.cs

```diff
@@ -92,6 +92,17 @@ public override object DeserializeObject(object value, Type type)
                 {
                     if (ReflectionUtils.GetTypeInfo(type).IsEnum)
                     {
+                        //first try to get all custom attributes
+                        var fields = type.GetRuntimeFields();
+                        foreach (var field in fields)
+                        {
+                            var attribute = (ParameterAttribute)field.GetCustomAttribute(typeof(ParameterAttribute));
+                            if (attribute != null)
+                            {
+                                if (attribute.Value.Equals(value))
+                                    return field.GetValue(null);
+                            }
+                        }
```

Comment:
We should cache the result of this in a dictionary of dictionaries (probably `Dictionary<Type, Dictionary<Enum, string>>`.) That way, the next time we try to serialize an enum value, we don't have to do this iteration. It'll be a very fast lookup.